summarize-internal-frag.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+# See summarize.py for the requirement on the input; the only exception is that
+# this script processes the table format of malloc stats because the json
+# format does not have the necessary information as of the time this script was
```

Comment:
I mean, we do now, right? We should just parse these out of json, rather than having a brittle manual parsing setup.